### PR TITLE
Correctly report error locations

### PR DESCRIPTION
**Pull request by SevInf**
Line is reported using 1-based indexing.
Column is reported instead of offset and does not gets reduced
when line is longer 80 characters.

@veged @arikon @indutny

### CODE REVIEW

File: lib/ometajs/core/grammar.js [hunk 1]
```diff
@@ -98,14 +99,15 @@ AbstractGrammar.prototype._getError = function getError(errback) {
           return false;
         }
 
-        offset = this._errorOffset - current;
+        colNumber = this._errorOffset - current + 1; //offsets are 0-based, columns are 1-based
```
Comment (indutny) [inline lib/ometajs/core/grammar.js]: 80 column limit.

File: lib/ometajs/core/grammar.js [hunk 2]
```diff
@@ -98,14 +99,15 @@ AbstractGrammar.prototype._getError = function getError(errback) {
...
         line = source;
-        lineNumber = i;
+        lineNumber = i + 1; //i is 0-based, line number is 1-based
```
Comment (indutny) [inline lib/ometajs/core/grammar.js]: Space after `//`

**Comment by indutny** [comment]
Looks good, may I ask you to provide a test?

**Comment by SevInf** [comment]
@indutny, done.

**Comment by indutny** [comment]
Thank you! Landed with very minor fixes in 89dd241 and 50ceb90

**Pull request closed without merging**